options/locale/locale_en-US.ini

```diff
@@ -1788,9 +1788,9 @@ pulls.unrelated_histories = Merge Failed: The merge head and base do not share a
 pulls.merge_out_of_date = Merge Failed: Whilst generating the merge, the base was updated. Hint: Try again.
 pulls.head_out_of_date = Merge Failed: Whilst generating the merge, the head was updated. Hint: Try again.
 pulls.has_merged = Failed: The pull request has been merged, you cannot merge again or change the target branch.
-pulls.push_rejected = Merge Failed: The push was rejected. Review the Git Hooks for this repository.
+pulls.push_rejected = Push Failed: The push was rejected. Review the Git Hooks for this repository.
 pulls.push_rejected_summary = Full Rejection Message
-pulls.push_rejected_no_message = Merge Failed: The push was rejected but there was no remote message.<br>Review the Git Hooks for this repository
+pulls.push_rejected_no_message = Push Failed: The push was rejected but there was no remote message.<br>Review the Git Hooks for this repository
```

Comment:
`<br>` doesn't seem to be correct for JSON error message. (well, an old problem .....)

![image](https://github.com/go-gitea/gitea/assets/2114189/4ffbe0d2-3c63-4171-b803-1346998eddfc)

Reply:
Fixed.